lib/utils/configTypes.ts

```diff
@@ -0,0 +1,273 @@
+        config.LOGS_BUCKET_NAME = fileConfig.LOGS_BUCKET_NAME || config.LOGS_BUCKET_NAME;
+        config.ACCESS_LOGS_BUCKET_NAME = fileConfig.ACCESS_LOGS_BUCKET_NAME || config.ACCESS_LOGS_BUCKET_NAME;
+        config.WEBSITE_BUCKET_NAME = fileConfig.WEBSITE_BUCKET_NAME || config.WEBSITE_BUCKET_NAME;
+        // Notebook settings
+        config.MLSPACE_LIFECYCLE_CONFIG_NAME = fileConfig.MLSPACE_LIFECYCLE_CONFIG_NAME || config.MLSPACE_LIFECYCLE_CONFIG_NAME;
+        config.NOTEBOOK_PARAMETERS_FILE_NAME = fileConfig.NOTEBOOK_PARAMETERS_FILE_NAME || config.NOTEBOOK_PARAMETERS_FILE_NAME;
+        // EMR settings
+        config.EMR_SECURITY_CONFIG_NAME = fileConfig.EMR_SECURITY_CONFIG_NAME || config.EMR_SECURITY_CONFIG_NAME;
+        // OIDC settings
+        config.IDP_ENDPOINT_SSM_PARAM = fileConfig.IDP_ENDPOINT_SSM_PARAM || config.IDP_ENDPOINT_SSM_PARAM;
+        config.INTERNAL_OIDC_URL = fileConfig.INTERNAL_OIDC_URL || config.INTERNAL_OIDC_URL;
+        config.OIDC_VERIFY_SSL = fileConfig.OIDC_VERIFY_SSL || config.OIDC_VERIFY_SSL;
+        config.OIDC_VERIFY_SIGNATURE = fileConfig.OIDC_VERIFY_SIGNATURE || config.OIDC_VERIFY_SIGNATURE;
+        config.OIDC_REDIRECT_URI = fileConfig.OIDC_REDIRECT_URI || config.OIDC_REDIRECT_URI;
+        // Other properties not prompted for in config-helper
+        config.SYSTEM_TAG = fileConfig.SYSTEM_TAG || config.SYSTEM_TAG;
+        config.IAM_RESOURCE_PREFIX = fileConfig.IAM_RESOURCE_PREFIX || config.IAM_RESOURCE_PREFIX;
+        config.APPLICATION_NAME = fileConfig.APPLICATION_NAME || config.APPLICATION_NAME;
+        config.PERMISSIONS_BOUNDARY_POLICY_NAME = fileConfig.PERMISSIONS_BOUNDARY_POLICY_NAME || config.PERMISSIONS_BOUNDARY_POLICY_NAME;
+        config.NOTIFICATION_DISTRO = fileConfig.NOTIFICATION_DISTRO || config.NOTIFICATION_DISTRO;
+        config.EXISTING_KMS_MASTER_KEY_ARN = fileConfig.EXISTING_KMS_MASTER_KEY_ARN || config.EXISTING_KMS_MASTER_KEY_ARN;
+        config.APIGATEWAY_CLOUDWATCH_ROLE_ARN = fileConfig.APIGATEWAY_CLOUDWATCH_ROLE_ARN || config.APIGATEWAY_CLOUDWATCH_ROLE_ARN;
+        config.COMMON_LAYER_ARN_PARAM = fileConfig.COMMON_LAYER_ARN_PARAM || config.COMMON_LAYER_ARN_PARAM;
+        config.ADDITIONAL_LAMBDA_ENVIRONMENT_VARS = fileConfig.ADDITIONAL_LAMBDA_ENVIRONMENT_VARS || config.ADDITIONAL_LAMBDA_ENVIRONMENT_VARS;
+        config.MANAGE_IAM_ROLES = fileConfig.MANAGE_IAM_ROLES || config.MANAGE_IAM_ROLES;
+        config.ENABLE_ACCESS_LOGGING = fileConfig.ENABLE_ACCESS_LOGGING || config.ENABLE_ACCESS_LOGGING;
+        config.CREATE_MLSPACE_CLOUDTRAIL_TRAIL = fileConfig.CREATE_MLSPACE_CLOUDTRAIL_TRAIL || config.CREATE_MLSPACE_CLOUDTRAIL_TRAIL;
+        config.ENABLE_TRANSLATE = fileConfig.ENABLE_TRANSLATE || config.ENABLE_TRANSLATE;
+        config.ENABLE_GROUNDTRUTH = fileConfig.ENABLE_GROUNDTRUTH || config.ENABLE_GROUNDTRUTH;
+        config.RESOURCE_TERMINATION_INTERVAL = fileConfig.RESOURCE_TERMINATION_INTERVAL || config.RESOURCE_TERMINATION_INTERVAL;
```

Comment:
Since these are all flat we can just replace all this with:
```suggestion
_.merge(config, fileConfig)
```

Reply:
Nice suggestion! Turns out you need to install `lodash` and import the module, but this is definitely a bit more concise than what I had.